src/main/java/org/wildfly/extension/elytron/FileSystemRealmDefinition.java

```diff
@@ -134,8 +134,8 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
 
             final int levels = LEVELS.resolveModelAttribute(context, model).asInt();
 
-            final String path = PATH.resolveModelAttribute(context, model).asString();
-            final String relativeTo = RELATIVE_TO.resolveModelAttribute(context, model).asString();
+            final String path = asStringIfDefined(context, PATH, model);
```

Comment:
I agree relative to needed to be changed as that attribute is optional, however path is mandatory so we don't need to wrap it with the asStringIfDefined call as by the time we reach this point it must always be defined.

Still checking the PR - this will not stop the merge though.